anax-in-container/Dockerfile_agbot.ubi

```diff
@@ -0,0 +1,50 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.0
+
+ENV ANAX_LOG_LEVEL 3
```

Comment:
Do we need all these env vars? They look like leftovers from testing that you did?

Reply:
Yes, it is needed by script/agbot_start.sh.  Here defines the default value. It can be overridden bu the user through `docker run -e ANAX_LOG_LEVEL=5 ...`.

Reply:
The other vars like DB_CONFIG, EXCHANGE_URL etc are the default value for other users to be able to bring the  container up and running. After discussing with Carl, we have decided define a minimal set of overridable env vars for other users.

Reply:
Ok, after more discussion with @cgiroua , I have removed all the ENVs from here.